goddard/schema.coffee

```diff
@@ -36,6 +36,15 @@ module.exports = exports = (app) ->
 	app.set('database', sequelize)
 
 	# set the models
+	Models.reports = sequelize.define('reports', {
+
+		nodeId: { type: Sequelize.INTEGER, field: 'nodeId' }
+		status: { type: Sequelize.STRING(255), field: 'status' }
+		message: { type: Sequelize.STRING(255), field: 'message' }
```

Comment:
part of the ORM. Will be *createdAt* and *updatedAt*. But will add the *timestamp* from the file too so you have it all.